UI.py

```diff
@@ -167,6 +167,24 @@ def CreateRetroactiveLeaveInsertionOptions():
     ]
     return retroactive_application_options
 
+def CreateIsMemberWorkingOptions():
+    member_options = [
+        create_option(
+            name = "discorduser",
+            description = "discord user",
+            option_type = SlashCommandOptionType.USER,
+            required = True
+        ),
+        create_option(
+            name = "date",
+            description = "date of interest",
+            option_type = SlashCommandOptionType.STRING,
+            required = True
+        )
```

Comment:
Let's not make this required and default to today

Reply:
Done